lib/DXIL/DxilShaderFlags.cpp

```diff
@@ -761,10 +780,11 @@ ShaderFlags ShaderFlags::CollectShaderFlags(const Function *F,
   flag.SetAdvancedTextureOps(hasAdvancedTextureOps);
   flag.SetWriteableMSAATextures(hasWriteableMSAATextures);
   flag.SetWaveMMA(hasWaveMMA);
-
+  flag.SetSampleCmpGradientOrBias(hasSampleCmpGradientOrBias);
```

Comment:
nit: Strange ordering, why not add the two lines together at the end?

Reply:
Fixed.